packages/gatsby-source-wordpress/src/steps/set-request-headers.ts

```diff
@@ -0,0 +1,25 @@
+import btoa from "btoa"
```

Comment:
do we need to import, can't we use buffer?

Reply:
Oh true! 👍